src/wallet/rpcwallet.cpp

```diff
@@ -1346,6 +1346,7 @@ void ListTransactions(const CWalletTx& wtx, const string& strAccount, int nMinDe
             entry.push_back(Pair("fee", ValueFromAmount(-nFee)));
             if (fLong)
                 WalletTxToJSON(wtx, entry);
+            entry.push_back(Pair("abandoned", wtx.isAbandoned()));
```

Comment:
Also it looks like you forgot to add the documentation [here](https://github.com/jonasschnelli/bitcoin/blob/df80f3d097984fa3b5b31be9ed86d2b489dcb5ec/src/wallet/rpcwallet.cpp#L1441)?